docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,453 @@ import { StyledContent } from '../../components/styled.elements';
 
 # Redocly configuration file
 
-The Redocly configuration file controls options for the OpenAPI CLI tool.
-It is also used across other Redocly products to help you control their behavior.
+The Redocly configuration file is used by OpenAPI CLI and other Redocly apps to control their behavior — from the strictness of the `lint` command, to how Redocly renders your docs.
 
-- Workflows uses it in the API registry to manage your APIs and control advanced features like region and link resolution.
-- Reference uses it to apply your custom settings when building API reference documentation (hosted and on-premise).
-- VS Code extension uses it for linting criteria, to apply custom settings for live documentation preview, and to set API definition root files.
+* Workflows uses it in the API registry to manage your APIs and control advanced features like region and link resolution.
+* Reference uses it to apply your custom settings when it's building API reference documentation (hosted and on-premise).
+* VS Code extension uses it for linting criteria, to apply custom settings for live documentation previews, and to set API definition root files.
 
-## Filename and location
+## Location, name and format
 
-The Redocly configuration file must be named `redocly.yaml` or `.redocly.yaml`, and placed into your working directory or the root of your project (depending on the Redocly product you're using).
+The Redocly configuration file must be named either `redocly.yaml` or `.redocly.yaml`. It must be located in the root of your project directory, and it must be in the YAML format. If you cloned the `openapi-starter` project, this is already set up for you.
 
-OpenAPI CLI will search for the configuration file in the working directory.
+All of the available sections and config options are documented below.
 
-:::warning
+## `organization`
 
-Multiple configuration files in the same directory are not allowed.
-If you have a `redocly.yaml` and a `.redocly.yaml` file  in the same directory, Redocly apps will display a warning in the output and ask you to choose one file to use.
+Stores your Redocly Workflows organization ID that the `push` command uses to push API definition files to the API registry. If an organization ID is not passed explicitly in command-line arguments, `push` will look for it in `redocly.yaml`.
 
+:::info
+In Redocly Workflows, organization ID and organization name are not the same thing.
 :::
 
-## File format and structure
+To find the organization ID:
+
+1. Log into Workflows.
+2. Access the **API registry** page.
+3. In your browser's address bar, find the page URL.
+4. The segment after `app.redocly.com/org/` is the organization ID.
+
+For example, if the URL is `app.redocly.com/org/test_docs`, the organization ID is `test_docs`.
+
+## `apis`
```

Comment:
This section is missing the property/type/description table summary (noticed that in other sections below).

patternProperties
{apiName}@{apiVersion}
{apiName}

Then, the api object:
root string
lint Lint object
openapi.features  OpenAPI Features object
etc...